bigbluebutton/bigbluebutton
# feat(plugin): Change enforced layout

### What does this PR do?

It allows the plug-in developer to change the enforcedLayout with a UI command.

See demo below:


https://github.com/user-attachments/assets/edf4c568-c379-4dfa-aee8-2aabf47ebb35


### How to test

Pretty simple: add the SampleActionButtonDropdown plugin to a meeting (the one in the correlated PR - see `more` section) and change the layout with whatever you prefer...;

### More

Closely related to https://github.com/bigbluebutton/bigbluebutton-html-plugin-sdk/pull/135
